lib/sidekiq-scheduler/extensions/web.rb

```diff
@@ -10,5 +10,5 @@
   Sidekiq::Web.use Rack::Static, urls: ['/stylesheets-scheduler'],
                                  root: ASSETS_PATH,
                                  cascade: true,
-                                 header_rules: [[:all, { 'Cache-Control' => 'public, max-age=86400' }]]
+                                 header_rules: [[:all, { 'cache-control' => 'public, max-age=86400' }]]
```

Comment:
Should this also have `, no-cache="Set-Cookie"`?

https://github.com/sidekiq/sidekiq/issues/5936

Reply:
It shouldn’t need this at all. Extensions should be able to add asset paths and reuse the default Rack::Static instance that Sidekiq installs.

Reply:
hmm interesting, I will check this when I have some time!